source/adapter.js

```diff
@@ -1036,6 +1040,14 @@ if ( (navigator.mozGetUserMedia ||
         ConstructSessionDescription(info.type, info.sdp);
     };
 
+    MediaStream = function (mediaStreamOrTracks) {
+      AdapterJS.WebRTCPlugin.WaitForPluginReady();
+      if (typeof mediaStreamOrTracks !== 'undefined' && mediaStreamOrTracks !== null) {
```

Comment:
What happens in the plugin if you receive undefined or null (might be different on IE and Safari) ?
Could we remove that if statement ?